src/utils/index.ts

```diff
@@ -52,8 +52,13 @@ export function generateGUID(): GUID {
 }
 
 /** Get an HTML element by id */
```

Comment:
could you expand on the doc a bit? Something like
```suggestion
/** 
 * Get an HTML element by id, looking inside the `root` (by default the whole `document`).
 *
 * The generic parameter `HTMLType` can be used to cast the element to a given type (e.g. `getById<HTMLInputElement>("foo")`) . The element type is not checked by this function.
 *
 * @throws if there is not element with the given id. 
 */
```